src/Symfony/Component/Process/Tests/ProcessTest.php

```diff
@@ -47,6 +47,25 @@ protected function tearDown()
         }
     }
 
+    /**
+     * @see https://github.com/symfony/symfony/issues/18249
```

Comment:
Usually we don't add such notes, we rely on the git history instead, should be removed.

Reply:
Removed.